fastapi_keycloak/__init__.py

```diff
@@ -0,0 +1,4 @@
+from fastapi_keycloak.api import FastAPIKeycloak
+from fastapi_keycloak.model import OIDCUser, UsernamePassword, HTTPMethod
+
+__all__ = [FastAPIKeycloak, OIDCUser, UsernamePassword, HTTPMethod]
```

Comment:
`__all__` has to contain stings (the names of the variables), so I would suggest to use the `__name__` property of the classes